cocos/scene-graph/node.ts

```diff
@@ -1447,13 +1459,36 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
      */
     public _static = false;
 
-    // world transform, don't access this directly
+    /**
+     * @engineInternal world transform, don't access this directly
+     */
+    public get w_pos () {
```

Comment:
There is already a worldPosition interface.

Reply:
we can't use worldPosition, because everytime we access worldPosition, engine called updateTransoform once,
the same as worldScale, worldRotation, worldMatrix

Reply:
ah, we have dirty checking